WalletWasabi.Tests/RegressionTests/CoinJoinTests.cs

```diff
@@ -156,7 +156,7 @@ public async Task CcjTestsAsync()
 		HttpRequestException httpRequestException = await Assert.ThrowsAsync<HttpRequestException>(async () => await CreateNewAliceClientAsync(roundId, registeredAddresses, signerPubKeys, requesters, inputsRequest));
 		Assert.Contains(HttpStatusCode.BadRequest.ToReasonString(), httpRequestException.Message);
 
-		byte[] dummySignature = new byte[65];
+		var dummySignature = new CompactSignature(0, new byte[65]);
```

Comment:
This new `CompactSignature` class is just a better way to express the signatures as one byte for the recovery byte plus the 64 bytes of the signature, what is what was stored in the `byte[65]` so, the even when the constructor doesn't validate in inputs we should pass a `new byte[64]`.

Reply:
Actually, this line fails as the second argument is actually being checked that it is `byte[64]`: https://github.com/MetacoSA/NBitcoin/commit/48341d508ef306dd7cfc1f50539acbd3b847ba23#diff-01894d0689ad40484dcab83b108730da1820e0c3dbf593b399d078ce084b6b50R16

Reply:
(https://github.com/MetacoSA/NBitcoin/commit/48341d508ef306dd7cfc1f50539acbd3b847ba23 is the commit that is relevant here)